src/app/Sankalan/Quiz/Quiz.php

```diff
@@ -21,29 +21,66 @@ protected function handleRoute()
     {
         $uri = $this->uri;
 
-        switch ($uri) {
-            case '':
-                $this->res->setContent('quiz home');
-                break;
-
-            case '/get-problems':
-                include 'get-problems.php';
-                $event = 'algoholics'; // figure out event name
-                $this->res = new JsonResponse(getProblems($event));
-                break;
-
-            default:
-                $event;
-                if (in_array($uri, $this->events)) {
-                    $event = $uri;
-                }
-                if (!isset($event)) {
-                    $this->res->setStatusCode(404);
-                    return;
-                }
-                $this->res->setContent('quiz page for : [' . $uri . ']');
-                break;
+        if (startsWith($uri, '/problems')) {
+            include 'problems.php';
```

Comment:
to maintain a general structure in project and for maintainability, we are using classes instead of including files directly. Operations related options to a feature can be kept in a single class